ajax/libs/vexflow/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "vexflow",
+  "filename": "vexflow-min.js",
+  "version": "1.2.84",
+  "description": "A JavaScript library for rendering music notation and guitar tablature",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/0xfe/vexflow.git"
+  },
+  "author": {
+    "name": "Mohit Muthanna Cheppudira",
+    "email": "mohit@muthanna.com",
+    "url": "http://muthanna.com"
+  },
+  "license": "MIT",
+  "homepage": "http://vexflow.com",
+  "keywords": [
+    "music",
+    "notation",
+    "guitar",
+    "tablature"
+  ],
+  "npmName": "vexflow",
+  "npmFileMap": [
+    {
+      "basePath": "releases",
+      "files": [
+        "vexflow*"
+      ]
+    }
```

Comment:
Please add the pattern to fetch the old versions v1.2.3 ~ v1.2.5 on npm. (You might need to check if we need all `vexflow*` under `lib`)
Like what https://github.com/cdnjs/cdnjs/pull/11439#issuecomment-312471744 did.